### PR TITLE
Update private-distributors-list.md

### PR DESCRIPTION
**Pull request by ericbannon**
Request to add Platform9 to the private distributors list

### CODE REVIEW

**Comment by ericbannon** [comment]
I signed it

**Comment by ericbannon** [comment]
Platform9 is already an organization on the CNFC and has signed the CLAhttps://identity.linuxfoundation.org/content/cncf-platform9-systems

I added my github email to my platform.com account through this project and it can be re-checked now 

I am authorized on that project

Thanks
Eric

File: security-release-process-documentation/private-distributors-list.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ could be in the form of the following:
 | stclair@google.com | Kubernetes PST |
 | upstream-security@heptio.com | Heptio |
 | vulnerabilityreports@cloudfoundry.org | Cloud Foundry |
+| security@platform9.com | Platform9
```
Comment (philips) [inline security-release-process-documentation/private-distributors-list.md]: @ericbannon you are missing a `|` at the end of the line.

**Comment by philips** [comment]
/lgtm
/hold

waiting for PST review of application

**Comment by ericbannon** [comment]
Any updates on this? Thanks guys
Eric

**Comment by philips** [comment]
Moved to https://github.com/kubernetes/security/pull/6

**Comment by ericbannon** [comment]
Thanks guys. We are in the middle of an ambitious sprint so I want to make sure our experts can help you guys in any way we can and be able to get access to key info. Appreciate all the help and look forward to being part of this group. Cheers

**Pull request closed without merging**